suse-open-source-policy.html

```diff
@@ -179,13 +179,23 @@ <h3 id="managing-source-code">Managing Source Code</h3>
 
     <h3 id="factory-first">Factory First</h3>
 
-    <p>When packaging new Open Source software or adding changes to existing packages follow the “Factory First” model. “Factory” is the common code stream all our distributions are based on. It is the immediate source for openSUSE Tumbleweed and it eventually ends up in openSUSE Leap and the SUSE Linux Enterprise distributions. This is a continuation of the “Upstream First” principle on the distribution level. It reduces maintenance effort and leverages the community.</p>
+    <p>When packaging new Open Source software or adding changes to existing packages follow the "Factory First" model. <a href="https://susedefines.suse.com/definition/opensuse-factory/">"Factory"</a> is the common code stream all our distributions are based on. It is the immediate source for openSUSE Tumbleweed and it eventually ends up in openSUSE Leap and the SUSE Linux Enterprise distributions. This is a continuation of the "Upstream First" principle on the distribution level. It reduces maintenance effort and leverages the community.</p>
 
-    <p>“Factory First” extends to other SUSE products. Packages for all products should generally come from the common code stream our distributions are based on. So for packages you need in any products contribute them to factory first.</p>
+    <p>"Factory First" extends to other SUSE products. Packages for all products should generally come from the common code stream our distributions are based on. So for packages you need in any products contribute them to factory first.</p>
+
+    <h3 id="first-summary">"First" summary</h3>
+
+    <p>"Upstream first" applies to all individual upstream projects, such as the Linux kernel and nearly any package on SLE.</p>
+
+    <p>"Upstream first" also applies when there is an estalished upstream distribution, such as Tumbleweed (for SUSE Linux Enterprise) or Uyuni (for SUSE Manager)</p>
+
+    <p>"Factory first" applies to the SUSE Linux Enterprise family. It is recommended for things tightly connected to SUSE Linux Enterprise. It should also be considered (not necessarily adopted) for every SUSE product.</p>
+
+    <p>Creating an "intermediate upstream" distribution in between an existing upstream project (like Ceph, Kubernetes or Cloud Foundry) and a SUSE product is not recommended for nor against. It should be assessed on a case by ase base without bias, taking benefits and costs into account.</p>
```

Comment:
I'm confused what this tries to say at all. if we just remove it, what is missing?

I'd just say to delete it right away.